src/referentielijsten/conf/base.py

```diff
@@ -390,89 +50,21 @@
 #
 # Django-Admin-Index
 #
-ADMIN_INDEX_SHOW_REMAINING_APPS = False
-ADMIN_INDEX_AUTO_CREATE_APP_GROUP = False
-ADMIN_INDEX_SHOW_REMAINING_APPS_TO_SUPERUSERS = True
 ADMIN_INDEX_DISPLAY_DROP_DOWN_MENU_CONDITION_FUNCTION = (
     "referentielijsten.utils.django_two_factor_auth.should_display_dropdown_menu"
 )
 
-#
-# DJANGO-AXES
-#
-AXES_CACHE = "axes"  # refers to CACHES setting
-# The number of login attempts allowed before a record is created for the
-# failed logins. Default: 3
-AXES_FAILURE_LIMIT = 10
-# If set, defines a period of inactivity after which old failed login attempts
-# will be forgotten. Can be set to a python timedelta object or an integer. If
-# an integer, will be interpreted as a number of hours. Default: None
-AXES_COOLOFF_TIME = 1
-# If set, specifies a template to render when a user is locked out. Template
-# receives cooloff_time and failure_limit as context variables. Default: None
-AXES_LOCKOUT_TEMPLATE = "account_blocked.html"
-AXES_LOCKOUT_PARAMETERS = [["ip_address", "user_agent", "username"]]
-# By default, Axes obfuscates values for formfields named "password", but the admin
-# interface login formfield name is "auth-password", so we obfuscate that as well
-AXES_SENSITIVE_PARAMETERS = ["password", "auth-password"]  # nosec
-
-# The default meta precedence order
-IPWARE_META_PRECEDENCE_ORDER = (
-    "HTTP_X_FORWARDED_FOR",
-    "X_FORWARDED_FOR",  # <client>, <proxy1>, <proxy2>
-    "HTTP_CLIENT_IP",
-    "HTTP_X_REAL_IP",
-    "HTTP_X_FORWARDED",
-    "HTTP_X_CLUSTER_CLIENT_IP",
-    "HTTP_FORWARDED_FOR",
-    "HTTP_FORWARDED",
-    "HTTP_VIA",
-    "REMOTE_ADDR",
-)
-
 #
 # MAYKIN-2FA
 #
 # It uses django-two-factor-auth under the hood so you can configure
 # those settings too.
 #
 # we run the admin site monkeypatch instead.
-TWO_FACTOR_PATCH_ADMIN = False
 # Relying Party name for WebAuthn (hardware tokens)
 TWO_FACTOR_WEBAUTHN_RP_NAME = "referentielijsten"
-# use platform for fingerprint readers etc., or remove the setting to allow any.
-# cross-platform would limit the options to devices like phones/yubikeys
-TWO_FACTOR_WEBAUTHN_AUTHENTICATOR_ATTACHMENT = "cross-platform"
-# add entries from AUTHENTICATION_BACKENDS that already enforce their own two-factor
-# auth, avoiding having some set up MFA again in the project.
-MAYKIN_2FA_ALLOW_MFA_BYPASS_BACKENDS = [
-    # "mozilla_django_oidc_db.backends.OIDCAuthenticationBackend",
```

Comment:
Commented out here but active in open-api-framework

Reply:
Does this affect anything?

Reply:
Good question. Does 2FA work for Referentielijsten?

Reply:
2FA does work for referentielijsten, and I'm not sure why it was commented out in the first place, because we always want to bypass 2FA when logging in with OIDC (because it's up to the identity providers to provide MFA)